hr_payroll_commission/README.rst

```diff
@@ -0,0 +1,90 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+Payroll Commissions
+===================
+
+This module allows you to add commission and reimbursement to payslip

```

Comment:
remove reimbursement, and detail about commission being calculated on received payments..